tools/pythonpkg/tests/fast/test_result.py

```diff
@@ -41,6 +41,10 @@ def test_result_timestamps(self, duckdb_cursor):
         assert rel.execute().fetchall() == [(datetime.datetime(2008, 1, 1, 0, 0, 11), datetime.datetime(2008, 1, 1, 0, 0, 1, 794000), datetime.datetime(2008, 1, 1, 0, 0, 1, 989260), datetime.datetime(2008, 1, 1, 0, 0, 1, 899268))]
 
     def test_result_interval(self):
+        try:
+            from pandas import DateOffset
+        except:
+            return
```

Comment:
I don't think we have in our CI a scenario where pandas can't be installed.
But when skipping a test please use `pytest.importorskip()`